### PR TITLE
Feat/challenges part 2 ex1

### PR DESCRIPTION
**Pull request by PedroPapa1**
1. **Smoker's Life Reduction Calculator**
   Write a program to calculate the reduction in a smoker's life expectancy. Ask for the number of cigarettes smoked per day and how many years the person has been smoking. Consider that a smoker loses 10 minutes of life for each cigarette. Calculate how many days of life a smoker will lose and display the total in days.

### CODE REVIEW

File: list2/ex1/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+const MINUTES_IN_A_DAY = 1440;
+const LIFE_LOST_PER_CIGARETTE_IN_MINUTES = 10;
+const DAYS_IN_A_YEAR = 365;
```
Comment (emiliosheinz) [inline list2/ex1/index.js:6]: Great usage of constants.

File: list2/ex1/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        const remainingDaysLost = Math.round(totalDaysLost % DAYS_IN_A_YEAR);
+
+        if (yearsLost > 0) {
+            console.log(`You lost ~${yearsLost} years and ~${remainingDaysLost} days of your life due to smoking.`);
+        } else {
+            console.log(`You lost ~${remainingDaysLost} days of your life due to smoking.`);
+        }
+    }
+
+    lifeLost();
```
Comment (emiliosheinz) [inline list2/ex1/index.js]: I don't think the `lifeLost` function is needed.

**Review by emiliosheinz — APPROVED** [review]

File: list2/ex1/index.js (hunk 1 above)
Comment (PedroPapa1) [reply list2/ex1/index.js:6]: Thanks!

File: list2/ex1/index.js (hunk 2 above)
Comment (PedroPapa1) [reply list2/ex1/index.js]: Ok I'll remove it, but why isn't it necessary?

Comment (emiliosheinz) [reply list2/ex1/index.js]: You are declaring it and calling it right after.

Comment (emiliosheinz) [reply list2/ex1/index.js]: Why is it needed?

image

Comment (PedroPapa1) [reply list2/ex1/index.js]: yeah you're right.

**Pull request merged**